contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,126 @@
+a Kubernetes maintainer gives the OK. We require PRs to be approved for testing
+in order to prevent people from running malicious code on our CI servers, for
+example mining for bitcoins.
+
+Usually someone will approve testing and review your PR within a day or two.
+Have some patience, of course; Kubernetes is a huge, semi-chaotic project and
+people are often busy debugging issues or preparing for the next release. If
+your PR languishes unattended for more than a couple days, @mention someone in
+the
+[OWNERS](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md)
+file of the your changes' root directory and you should get a quicker response.
+
+In order for your PR to be merged, you need four things. When you have all four,
+the @k8s-merge-robot will automatically merge your commits.
+
+ 1. CNCF-CLA verification. You must sign the Contributor License Agreement to
+    contribute to Kubernetes. Check out our FAQ at
+    [https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ](https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ).
+    Once you sign the CLA, @k8s-ci-robot should automatically add the "cncf-cla:
+    yes" label.
+ 2. The "approved" label. Get this by having an owner from the OWNERS file
+    comment "/approve" on your PR.
+ 3. The LGTM label. Get this by having a Kubernetes org member write "/lgtm" on
+    your PR.
+ 4. All green CI. If CI fails, the robots will comment with instructions on how
+    to rerun the tests.
+
+There is an entire SIG
+([sig-contributor-experience](https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ))
+devoted to improving your experience as a contributor. Contributing to
+Kubernetes should be easy. If you find a rough edge, let us know!  File an issue
+in the appropriate repository if you know it, send a message on [our Slack
+channel](https://kubernetes.slack.com/messages/C1TU9EB9S/details) or to the
+[owners](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md)
+of contributor-experience. Better yet, help us fix it by joining the SIG; just
+show up to one of the [bi-weekly
+meetings](https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/edit).
+
+The Release Process and Code Freeze
+-----------------------------------
+
+Every so often @k8s-merge-robot will refuse to merge your PR, saying something
+about release milestones. This happens when we are in a code freeze for a
+release. In order to ensure Kubernetes is stable, we stop merging everything
+that's not a bugfix, then focus on making all the release tests pass. This code
+freeze usually lasts two weeks and happens once per quarter.
+
+If you're new to Kubernetes, you won't have to worry about this too much. After
+you've contributed for a few months, you will likely [become an official SIG
+member](https://github.com/kubernetes/community/blob/master/community-membership.md)
+and take ownership of some of the release tests. At this point, you'll work with
```

Comment:
There is test ownership which is sig based, but also release tests which are maintained by sig-release.